ember-app/app/app.js

```diff
@@ -25,7 +25,6 @@ var HuBoard = Ember.Application.extend({
   modulePrefix: config.modulePrefix,
   podModulePrefix: config.podModulePrefix,
   Resolver: Resolver,
-  rootElement: "#application",
```

Comment:
body by default I think, part of the redesign of the layout I switched it to use the body